.eslintrc.js

```diff
@@ -67,11 +67,12 @@ module.exports = {
     },
     {
       // Flow specific rules
-      files: ['*.js.flow', '*/*flow.js'],
+      files: ['*.js.flow', '*/*flow.js', '*.flow.test.js'],
       extends: ['plugin:flowtype/recommended'],
       plugins: ['flowtype'],
       rules: {
         'flowtype/generic-spacing': ['off'],
+        'no-unused-vars': ['off'],
```

Comment:
I just wonder why would we need unused vars?

Reply:
This is only allowed for flow typed files, and since the application is written in TypeScript, this rule will only apply to flow type definitions or tests. It is perfectly valid to have unused variables in these scenarios.